### PR TITLE
Replaces 'address' with 'device' in UI strings

### LINKED ISSUE #7374: Emphasize "Network address" to "Device" in copy for facility sync workflows
### Observed behavior

Some background from @jamalex : "Devices" are Kolibri peer servers that have a permanent ID (`instance_id` in Morango and other places) and might have a more transient URL/protocol.

The current UIs emphasize the _address_ as the main entity, instead of the _device_

### Expected behavior

* UI copy for p2p-facility-sync flows emphasizes "Device" over "Address"
* (Bonus) the code implementing this stuff possibly be renamed for DX purposes

### Needs Clarification

* What headers and non-header copy needs to be changed?
* Do we still retain the original "Network address" copy for import/export flows?
* Does "Device" work as a unifying term for both Kolibri Peers + Studio/Content sources so it can be used for both facility-sync + content import/export flows?

### Errors and logs

…

### Steps to reproduce

…

### Context

…

### PR DESCRIPTION
**Pull request by bjester**
## Summary

- Replaces usage of 'address' in UI strings with 'device'
- Renames components for code clarity to also refer to 'device'
- Updates usages of renamed components with minor organization updates

## References

Resolves https://github.com/learningequality/kolibri/issues/7374

## Reviewer guidance

- This PR does not intend to change functionality, only to update strings and rename code for clarity
- `yarn run build` completed successfully and server started successfully

----

## Testing checklist

- [X] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [X] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/assets/src/mixins/commonSyncElements.js [hunk 1]
```diff
@@ -11,18 +11,18 @@ const syncStrings = createTranslator('CommonSyncStrings', {
     context: 'Title of menu where the user selects the source from where to import a facility',
   },
   selectNetworkAddressTitle: {
-    message: 'Select network address',
+    message: 'Select network device',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/mixins/commonSyncElements.js]:
```suggestion
    message: 'Select device',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue [hunk 2]
```diff
@@ -150,7 +150,7 @@
         context: 'Example of a network address.',
       },
       errorCouldNotConnect: {
-        message: 'Could not connect to this network address',
+        message: 'Could not connect to this network device',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue]:
```suggestion
        message: 'Could not connect to this device',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue [hunk 3]
```diff
@@ -314,44 +314,44 @@
...
-      forgetAddressButtonLabel: {
+      forgetDeviceButtonLabel: {
         message: 'Remove',
         context:
-          'Removes a network address from the list of network addresses which have been registered in the Device > Facilities section.',
+          'Removes a network device from the list of network devices which have been registered in the Device > Facilities section.',
       },
       header: {
-        message: 'Select network address',
+        message: 'Select network device',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue]:
```suggestion
        message: 'Select device',
```

File: kolibri/core/assets/src/mixins/commonSyncElements.js [hunk 4]
```diff
@@ -11,18 +11,18 @@ const syncStrings = createTranslator('CommonSyncStrings', {
...
+    message: 'New device',
     context:
-      'Title of the menu where the user manually adds a new device address from where to import a facility',
+      'Title of the menu where the user manually adds a new device device from where to import a facility',
   },
   addNewAddressAction: {
-    message: 'Add new address',
-    context: 'Label for a button that opens a menu to save a new network address.',
+    message: 'Add new device',
+    context: 'Label for a button that opens a menu to save a new network device.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/mixins/commonSyncElements.js]:
```suggestion
    context: 'Label for a button that opens a menu to save a new device.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue [hunk 5]
```diff
@@ -150,7 +150,7 @@
...
         context:
           "This is an error message that an admin will see when Kolibri can't connect to another network device when trying to import resources.",
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue]:
```suggestion
          "This is an error message that an admin will see when Kolibri can't connect to another device when trying to import resources.",
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue [hunk 6]
```diff
@@ -160,14 +160,14 @@
           'This is an error validation message that an admin will see when they do not enter a valid network address.',
       },
       header: {
-        message: 'New address',
+        message: 'New device',
         context:
-          'The title of the section that an admin accesses when they select the "Add new address" link in the Device > Facilities section.\n\nThey use this screen to add a new network address.',
+          'The title of the section that an admin accesses when they select the "Add new device" link in the Device > Facilities section.\n\nThey use this screen to add a new network device.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue]:
```suggestion
          'The title of the section that an admin accesses when they select the "Add new device" link in the Device > Facilities section.\n\nThey use this screen to add a new device.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue [hunk 7]
```diff
@@ -160,14 +160,14 @@
...
       },
       nameDesc: {
-        message: 'Choose a name for this address so you can remember it later:',
+        message: 'Choose a name for this device so you can remember it later:',
         context:
-          'When an admin adds a new network address they can give it a name. This is a helper text to remind them to choose a name for the address.',
+          'When an admin adds a new network device they can give it a name. This is a helper text to remind them to choose a name for the device.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue]:
```suggestion
          'When an admin adds a new device they can give it a name. This is a helper text to remind them to choose a name for the device.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue [hunk 8]
```diff
@@ -179,12 +179,12 @@
       },
       submitButtonLabel: {
         message: 'Add',
-        context: 'Text for button used to add a new network address.',
+        context: 'Text for button used to add a new network device.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue]:
```suggestion
        context: 'Text for button used to add a new device.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue [hunk 9]
```diff
@@ -179,12 +179,12 @@
...
       },
       tryingToConnect: {
         message: 'Trying to connect to server…',
         context:
-          'Progress message an admin sees when Kolibri is attempting to connect to a network address.',
+          'Progress message an admin sees when Kolibri is attempting to connect to a network device.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/AddDeviceForm.vue]:
```suggestion
          'Progress message an admin sees when Kolibri is attempting to connect to a device.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue [hunk 10]
```diff
@@ -314,44 +314,44 @@
     },
     $trs: {
       deletingFailedText: {
-        message: 'There was a problem removing this address',
+        message: 'There was a problem removing this device',
         context:
-          'Error message that displays when an admin attempts to remove a network address, but is unable to do so.',
+          'Error message that displays when an admin attempts to remove a network device, but is unable to do so.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue]:
```suggestion
          'Error message that displays when an admin attempts to remove a device, but is unable to do so.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue [hunk 11]
```diff
@@ -314,44 +314,44 @@
...
       },
       fetchingFailedText: {
-        message: 'There was a problem getting the available addresses',
+        message: 'There was a problem getting the available devices',
         context:
-          'Error message that displays when an admin attempts to find a network address, but the address is not found.',
+          'Error message that displays when an admin attempts to find a network device, but the device is not found.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue]:
```suggestion
          'Error message that displays when an admin attempts to find a device, but the device is not found.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue [hunk 12]
```diff
@@ -314,44 +314,44 @@
...
       },
-      forgetAddressButtonLabel: {
+      forgetDeviceButtonLabel: {
         message: 'Remove',
         context:
-          'Removes a network address from the list of network addresses which have been registered in the Device > Facilities section.',
+          'Removes a network device from the list of network devices which have been registered in the Device > Facilities section.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue]:
```suggestion
          'Removes a device from the list of devices which have been registered in the Device > Facilities section.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue [hunk 13]
```diff
@@ -314,44 +314,44 @@
...
       },
       header: {
-        message: 'Select network address',
+        message: 'Select network device',
         context:
-          "In the Device > Facilities section, you select the 'SYNC' option to choose the network address you want to sync from.\n\nYou do this in the 'Select network address' section which displays a list of network addresses.",
+          "In the Device > Facilities section, you select the 'SYNC' option to choose the network device you want to sync from.\n\nYou do this in the 'Select network device' section which displays a list of network devices.",
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue]:
```suggestion
          "In the Device > Facilities section, you select the 'SYNC' option to choose the device you want to sync from.\n\nYou do this in the 'Select device' section which displays a list of devices.",
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue [hunk 14]
```diff
@@ -314,44 +314,44 @@
...
       },
-      newAddressButtonLabel: {
-        message: 'Add new address',
+      newDeviceButtonLabel: {
+        message: 'Add new device',
         context:
-          'The "Add new address" link appears in the \'Select network address\' screen. This option allows you to add a new network address from which to sync data.',
+          'The "Add new device" link appears in the \'Select network device\' screen. This option allows you to add a new network device from which to sync data.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue]:
```suggestion
          'The "Add new device" link appears in the \'Select device\' screen. This option allows you to add a new device from which to sync data.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue [hunk 15]
```diff
@@ -314,44 +314,44 @@
...
       },
       lodSubHeader: {
         message: 'Select a device with Kolibri version 0.15 to import learner user accounts',
         context:
-          "In the first startup wizard, when you select to 'Import one or more user accounts from an existing facility' option to choose the network address you want to sync from.\n\nYou do this in the 'Select network address' section which displays a list of network addresses.",
+          "In the first startup wizard, when you select to 'Import one or more user accounts from an existing facility' option to choose the network device you want to sync from.\n\nYou do this in the 'Select network device' section which displays a list of network devices.",
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue]:
```suggestion
          "In the first startup wizard, when you select to 'Import one or more user accounts from an existing facility' option to choose the device you want to sync from.\n\nYou do this in the 'Select device' section which displays a list of devices.",
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue [hunk 16]
```diff
@@ -314,44 +314,44 @@
...
       },
-      noAddressText: {
-        message: 'There are no addresses yet',
+      noDeviceText: {
+        message: 'There are no devices yet',
         context:
-          "This message displays when there are no network addresses to sync with.\n\nIt appears when selecting 'SYNC' in the Device > Facilities section if there are no network addresses.",
+          "This message displays when there are no network devices to sync with.\n\nIt appears when selecting 'SYNC' in the Device > Facilities section if there are no network devices.",
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue]:
```suggestion
          "This message displays when there are no devices to sync with.\n\nIt appears when selecting 'SYNC' in the Device > Facilities section if there are no devices.",
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue [hunk 17]
```diff
@@ -314,44 +314,44 @@
...
       },
-      refreshAddressesButtonLabel: {
-        message: 'Refresh addresses',
+      refreshDevicesButtonLabel: {
+        message: 'Refresh devices',
         context:
-          'This message displays if there was a problem getting the network addresses. It allows the user to refresh the application to be able to see all the addresses available.',
+          'This message displays if there was a problem getting the network devices. It allows the user to refresh the application to be able to see all the devices available.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue]:
```suggestion
          'This message displays if there was a problem getting the devices. It allows the user to refresh the application to be able to see all the devices available.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/index.vue [hunk 18]
```diff
@@ -93,13 +93,13 @@
       },
     },
     $trs: {
-      addAddressSnackbarText: {
-        message: 'Successfully added address',
-        context: 'This message appears if a network address has been added correctly.',
+      addDeviceSnackbarText: {
+        message: 'Successfully added device',
+        context: 'This message appears if a network device has been added correctly.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/index.vue]:
```suggestion
        context: 'This message appears if a device has been added correctly.',
```

File: kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/index.vue [hunk 19]
```diff
@@ -93,13 +93,13 @@
...
       },
-      removeAddressSnackbarText: {
-        message: 'Successfully removed address',
-        context: 'This message appears if a network address has been removed correctly.',
+      removeDeviceSnackbarText: {
+        message: 'Successfully removed device',
+        context: 'This message appears if a network device has been removed correctly.',
```
Comment (marcellamaki) [inline kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/index.vue]:
```suggestion
        context: 'This message appears if a device has been removed correctly.',
```

File: kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue [hunk 20]
```diff
@@ -245,9 +245,9 @@
       },
     },
     $trs: {
-      addAddressSnackbarText: {
-        message: 'Successfully added address',
-        context: 'This message appears if a network address has been added correctly.',
+      addDeviceSnackbarText: {
+        message: 'Successfully added device',
+        context: 'This message appears if a network device has been added correctly.',
```
Comment (marcellamaki) [inline kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue]:
```suggestion
        context: 'This message appears if a device has been added correctly.',
```

File: kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue [hunk 21]
```diff
@@ -258,15 +258,15 @@
         context:
           'This text appears next to the "Add new address" link. This option allows you to add a new network address from which to sync data.',
       },
-      newAddressButtonLabel: {
-        message: 'Add new address',
+      newDeviceButtonLabel: {
+        message: 'Add new device',
         context:
-          'The "Add new address" link appears in the \'Select network address\' screen. This option allows you to add a new network address from which to sync data.',
+          'The "Add new device" link appears in the \'Select network device\' screen. This option allows you to add a new network device from which to sync data.',
```
Comment (marcellamaki) [inline kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue]:
```suggestion
          'The "Add new device" link appears in the \'Select device\' screen. This option allows you to add a new device from which to sync data.',
```

**Review by marcellamaki — COMMENTED** [review]
Thanks @bjester - I've made "suggested changes" because following the string review today, the conclusion from the team was to update the UI-facing text to just use "device" rather than "network device".  I've included the changes to the context as well for consistency for the translators.

@radinamatic would appreciate a proofread from you to confirm I have not missed any. Then these can be easily updated within the PR :)

**Review by radinamatic — APPROVED** [review]
All strings are now as per the outcome of the final string review! 👍🏽 💯 :shipit:

**Pull request merged**